qdrant-landing/content/documentation/interfaces.md

```diff
@@ -62,7 +62,7 @@ First, run the Docker container:
 
 ```bash
 docker run -p 6333:6333 -p 6334:6334 \
-    -v $(pwd)/qdrant_storage:/qdrant/storage \
+    -v $(pwd)/qdrant_storage:/qdrant/storage:z \
```

Comment:
What does `:z` mean?

Reply:
It prevents SELinux errors when trying to write and access host disk space. I ran into this error when trying to start the container on Fedora. 
https://jaosorior.dev/2018/selinux-and-docker-notes/

|"z(lower): relabels the content you’re mounting into the container, and makes it shareable between containers."